contrib/test.sh

```diff
@@ -4,12 +4,27 @@ set -ex
 
 CRATES="bitcoin hashes internals fuzz"
 
-for crate in ${CRATES}
+for dep in $deps
```

Comment:
`$deps` is not defined.

Reply:
face palm, thanks man. I added `DEPS="recent minimal"`, and put quotes around `$dep` everywhere as suggested by `shellcheck`.